### ISSUE TITLE
Use the conventional origin name instead of github as a remote name

### ISSUE THREAD

**Issue by fbdtemme** [lizard-bio/cookiecutter-analytical-project#8]
Is there a specific reason we are not using the standard `origin` in the git repo setup instructions?

```shell
cd <your-project-name>
git init
git remote add github git@github.com:lizard-bio/<your-project-name>.git
git branch -m main
git add --all
git commit -m "first commit"
git push -f github main
```

Using the common `origin` as upstream name makes it much easier to follow different git instructions from online guides and resources.

**Comment by erikvdp** [comment]
Not really, I find that it improves understanding what happens for some people when using dedicate remote names, esp when there are multiple remotes. Fine for me to switch to origin

**Comment by picousse** [comment]
Fixed: https://github.com/lizard-bio/cookiecutter-analytical-project/pull/25

Can you review?

**Comment by picousse** [comment]
fixed
